.travis.yml

```diff
@@ -38,11 +38,12 @@ matrix:
 
 before_script:
   - if [[ $PREFER_LOWEST != 1 ]]; then composer install --prefer-source --no-interaction ; fi
-  - if [[ $PREFER_LOWEST == 1 ]]; then composer require --dev dereuromark/composer-prefer-lowest; fi
-  - if [[ $PREFER_LOWEST == 1 ]]; then composer update --prefer-dist --no-interaction --prefer-lowest --prefer-stable ; fi
 
   - if [[ $CHECKS != 1 ]]; then composer require phpunit/phpunit:"^8.0.4"; fi
 
+  - if [[ $PREFER_LOWEST == 1 ]]; then composer require --dev dereuromark/composer-prefer-lowest; fi
+  - if [[ $PREFER_LOWEST == 1 ]]; then composer update --prefer-dist --no-interaction --prefer-lowest --prefer-stable ; fi
+
```

Comment:
This block needs to be moved to back line 41

Reply:
I think this is now functional and gives the correct order to work for all combinations in the variables matrix.